.storybook/styles.less

```diff
@@ -0,0 +1,11 @@
+@import "../src/core-components/styles";
+@import "../src/assets/styles/themes/light";
+@import "../src/assets/styles/themes/dark";
+
+.dark, .light {
+  background: var(--ergo-default-background);
```

Comment:
The color on the light theme is wrong. Design layouts use #fafafa (aka `@light-color-gray-2`)